DoSomething/gladiator
# Gateway Authentication of Users

#### What's this PR do?
Allow admins to authenticate via Northstar using Gateway.

#### How should this be manually tested?
1. Clicking 'Log In' redirects user to Northstar authentication portal. 
2. If Admin, user should be able to sign in to Gladiator and see all contests. 
3. If Staff, user should also be able to sign in to Gladiator and see all contests. 
4. If unauthorized user tries to log in, all error handled in Northstar authentication portal. 

#### Any background context you want to provide?
Gateway RestApiClient not implemented yet, so user activities currently crash. 

#### What are the relevant tickets?
Ref #181
